docs/go-dependencies.md

```diff
@@ -0,0 +1,66 @@
+# Managing go dependencies
+
+Go dependencies are managed using
+the [godep tool](https://github.com/tools/godep). `build/dep.sh` is an ease of
+use wrapper around `godep`.
+
+```text
+dep.sh COMMAND ...
+
+Manages godep dependencies. This mostly a wrapper around operations
+involving godep and a Docker image containing a clean copy of the
+dependencies.
+
+image
+  Create image with godep dependencies (k8s-dns-godep).
+
+verify
+  Verify that the godep matches the dependencies in the code. Exits
+  with success if there are no differences.
+
+enter [-u]
+  Run Docker container interactively to manage godep. If -u is
+  specified, then update the image with changes made.
+
+save [PKG1 PKG2 ...]
+  save PKGs as new dependencies. Note: the PKGs must be referenced
+  from the code, otherwise godep will ignore the new packages.
+
+-h|--help
+  This help message.
+
+USAGE
+
+Add a new package:
```

Comment:
Why adding a new package needs not be run in container, but updating a package need to be in container?